src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -295,4 +295,32 @@ public enum StreamMetadataKey
         /// </summary>
         Year
     }
+
+    /// <summary>
+    /// Enumeration of audio extract option.
+    /// </summary>
+    /// <seealso cref="Player.EnableExportingAudioData"/>
+    /// <since_tizen> 6 </since_tizen>
+    public enum PlayerAudioExtractOption
+    {
+        /// <summary>
+        /// Sync with the playback clock and multichannel audio stream
```

Comment:
Suggestion: 
/// Sync the playback clock and multi-channel audio stream.
or
/// Sync multi-channel audio stream with the playback clock.

Reply:
https://github.com/Samsung/TizenFX/pull/955
i've updated it